src/Sylius/Behat/Client/RequestInterface.php

```diff
@@ -56,4 +56,6 @@ public function updateFiles(array $newFiles): void;
     public function addSubResource(string $key, array $subResource): void;
 
     public function removeSubResource(string $subResource, string $id): void;
+
+    public function authorize(string $token): void;
```

Comment:
What is the reason for extracting this method?

Reply:
I need to make a request without authorization, therefore the previous `create` method was no-go for me. However, the rest of the methods still needs to authorize so IMHO it is better to provide it as a part of Public API. BTW, I would like completely decouple Request creation from APIClient and I see these changes as a first steps in this direction.